Strategy11/formidable-forms
# Add Missing Animation to Field Group Message Element

This PR addresses the omission of animation for the Field Group Message Element. Instead of using JavaScript for enqueuing the necessary CSS for animations, we have transitioned to a PHP-based approach for improved performance and consistency.

## Related Issue:
[Missing Animation for Field Group Message Element After Recent Dashboard UI Update #4758](https://github.com/Strategy11/formidable-pro/issues/4758)

## Testing Instructions:
To ensure the animation has been successfully integrated and functions as expected, please follow these steps:

1. **Form Builder:**
   - Open the form builder.
   - Select any field to trigger the bottom banner.
   - Confirm that the bottom banner appears with the intended animation.

2. **General Animation Check:**
   - Navigate to the Dashboard and Form Templates pages, and also check the Overlay element.
   - Observe the UI to confirm that animations are present and functioning correctly.

## Output Comparison:
### Before
![2024-01-30 11 34 39](https://github.com/Strategy11/formidable-pro/assets/69119241/0842ac93-928e-4ab4-8900-6deb4c819fbb)

### After
![2024-01-30 11 32 47](https://github.com/Strategy11/formidable-pro/assets/69119241/572cfecd-c793-4e7d-addf-ce12a6c522e0)
